{"uuid": "a897d926-9346-43ad-a3a8-db8a353c5d43", "children": ["4310145e-394b-490a-8490-5749eb6ff3e3"], "befores": [{"name": "app", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status\nfrom disconnected: received Inspector.detached event\n  (Session info: chrome=102.0.5005.61)\nStacktrace:\n#0 0x55b8b7c17f33 <unknown>\n#1 0x55b8b7962118 <unknown>\n#2 0x55b8b794dff5 <unknown>\n#3 0x55b8b794da55 <unknown>\n#4 0x55b8b794cf15 <unknown>\n#5 0x55b8b794bc0f <unknown>\n#6 0x55b8b794c19c <unknown>\n#7 0x55b8b795bba5 <unknown>\n#8 0x55b8b794d7a6 <unknown>\n#9 0x55b8b794e6bb <unknown>\n#10 0x55b8b794da72 <unknown>\n#11 0x55b8b794cf15 <unknown>\n#12 0x55b8b794bc0f <unknown>\n#13 0x55b8b794c042 <unknown>\n#14 0x55b8b7963b92 <unknown>\n#15 0x55b8b79c9a3f <unknown>\n#16 0x55b8b79b5f72 <unknown>\n#17 0x55b8b79c92e4 <unknown>\n#18 0x55b8b79b5e63 <unknown>\n#19 0x55b8b798b82a <unknown>\n#20 0x55b8b798c985 <unknown>\n#21 0x55b8b7c5c4cd <unknown>\n#22 0x55b8b7c605ec <unknown>\n#23 0x55b8b7c4671e <unknown>\n#24 0x55b8b7c61238 <unknown>\n#25 0x55b8b7c3b870 <unknown>\n#26 0x55b8b7c7d608 <unknown>\n#27 0x55b8b7c7d788 <unknown>\n#28 0x55b8b7c97f1d <unknown>\n#29 0x7faee6868609 <unknown>\n\n", "trace": "  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/user/PycharmProjects/demoaq_test/tests/conftest.py\", line 63, in app\n    browser.open_url(base_url).driver.set_window_size(width=1980, height=1280)\n  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/selene/support/shared/browser.py\", line 204, in open_url\n    return self.open(absolute_or_relative_url)\n  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/selene/support/shared/browser.py\", line 78, in open\n    self.config.get_or_create_driver().get(url)\n  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 442, in get\n    self.execute(Command.GET, {'url': url})\n  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 430, in execute\n    self.error_handler.check_response(response)\n  File \"/home/user/.virtualenvs/demoaq_test/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1659391854719, "stop": 1659391894837}], "start": 1659391854719, "stop": 1659391894962}